config/initializers/devise.rb

```diff
@@ -0,0 +1,259 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
```

Comment:
I think it's fine to leave these comments since they are generated, and explaining all the options which you'd have to look up otherwise